src/Z38/SwissPayment/StructuredPostalAddress.php

```diff
@@ -43,11 +43,31 @@ class StructuredPostalAddress implements PostalAddressInterface
      */
     public function __construct($street, $buildingNo, $postCode, $town, $country = 'CH')
```

Comment:
As the library is targeted towards Switzerland, I will keep the default argument for now. I will keep your point in mind in case we extend the scope of the library.